packages/edit-post/src/components/layout/style.scss

```diff
@@ -30,7 +30,8 @@
 @include editor-left(".edit-post-layout__content .components-editor-notices__snackbar");
```

Comment:
I'm assuming this mixin is not needed anymore if you want to center them

Reply:
👍 Definitely had no clue what this was for. 😜